events_protocol/core/builder.py

```diff
@@ -0,0 +1,50 @@
+import typing
+from uuid import uuid4
+
+from events_protocol.core.exception import EventException, EventNotFoundException
+from events_protocol.core.logging import JsonLogger
+from events_protocol.core.model.event import (
+    Event,
+    EventMessage,
+    PayloadType,
+    ResponseEvent,
+)
+from events_protocol.core.model.event_type import EventErrorType, EventSuccessType
+from events_protocol.core.logging.mixins.loggable import LoggableMixin
+
+
+class EventBuilder(LoggableMixin):
+    @classmethod
+    def error_for(
+        cls,
+        exc: typing.Union[EventException],
```

Comment:
`exc` também pode simbolizar `execution`
```suggestion
        exception: typing.Union[EventException],
```

Reply:
Alterada todas as exceptions do projeto  de `exc` para `exception`